apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -63,14 +63,14 @@ class CensusForm extends Component {
   checkShowFollowUp() {
     const twentyHours = this.state.submission.twentyHours;
     this.setState({
-      showFollowUp: (twentyHours === 'some' || twentyHours === 'all')
+      showFollowUp: (twentyHours === 'SOME' || twentyHours === 'ALL')
     }, this.checkShowPledge);
   }
```

Comment:
Looks like for HoC signup we also show the follow up questions if `tenHours === "ALL"`. Is that also the case for /yourschool?

Reply:
They actually have different checks. I'm already changing so much here that I didn't want to change this behavior also.